requirements.txt

```diff
@@ -1,18 +1,26 @@
-colorama
-demjson3
+colorama~=0.4.6
+demjson3~=3.0.6
 faiss-cpu
 geopy
-numpy
-openai
+numpy~=1.26.1
+openai~=0.28.1
 pybars3
 pymupdf
-python-dotenv
-pytz
-questionary
-rich
-tiktoken
+python-dotenv~=1.0.0
+pytz~=2023.3.post1
+questionary~=2.0.1
+rich~=13.6.0
+tiktoken~=0.5.1
 timezonefinder
-tqdm
-typer
+tqdm~=4.66.1
+typer~=0.9.0
 llama_index
-prettytable
+prettytable~=3.9.0
+requests~=2.31.0
+aiohttp~=3.8.6
+SQLAlchemy~=2.0.22
+fitz>=0.0.1.dev2

```

Comment:
It had to be "newer" than just adding "fitz" without a version requirement or id could not find the open function. Not sure why that works as it does